project/CrossJava.scala

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+// this is copy/pasted from https://github.com/akka/akka/blob/5576c233d063b3ee4cfc05d8e73c614a3dea478d/project/CrossJava.scalas
+
+package akka
+
+import java.io.File
+
+import scala.annotation.tailrec
+import scala.collection.immutable.ListMap
+import sbt._
+import sbt.librarymanagement.SemanticSelector
+import sbt.librarymanagement.VersionNumber
+import akka.CrossJava.nullBlank
+
+/*
+ * Tools for discovering different Java versions,
+ * will be in sbt 1.3.0 (https://github.com/sbt/sbt/pull/4139 et al)
```

Comment:
Can we chop this down?  We don't use most of it, and let's use what's in sbt 1.3 + the little more we need.

Reply:
Sure! I have https://github.com/lagom/lagom/issues/2553 but ATM I'm trying to keep things as identical as possible.

Reply:
OK.  You can just symlink it instead, then, to avoid another fork.